American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,96 @@
+
+---
+
+# Questioning the Future: American Elections and Crypto from a Cypherpunk Perspective
+
+The American elections of 2024 may be the most important event of the year. Not just to Americans, but to the whole world, since the US. remains the greatest economic and cultural power in our world. Scheduled for November 5, voters will determine both their legislative representatives and the future occupant of the presidency. The outcome of these elections is poised to reverberate across international boundaries, shaping the trajectory of global affairs for years to come.
+
+It appears that both the Republican and Democratic parties have solidified their contenders. With all indicators pointing towards a bid from the incumbent President Joe Biden representing the Democrats and the likelihood of former President Donald Trump stepping into the ring as the Republican nominee.
+
+Will the elections influence the crypto scene? Could crypto benefit from any candidate? And what does this mean for individuals deeply entrenched in the cypherpunk movement? Let's go through these questions!
+
+## Biden administration view on crypto
+
+The Biden administration was marked by a period of regulatory siege against the crypto market. These attacks were mainly coordinated by the Securities and Exchange Commission (SEC), which has Gary Gensler as its chairman, and increased after the bankruptcies of custodial service providers FTX, BlockFi, Voyager Digital, and other platforms.
+
+In 2023 alone the SEC took 26 enforcement actions targeting the crypto industry. The motives for these many initiatives were: to prevent fraud, reduce market manipulation, and increase control over exchanges. Those actions were the result of President Biden’s March 9 Executive Order (EO) on Ensuring Responsible Development of Digital Assets.
+
+The White House released this document on  March 9th, revealing the stance of Biden's administration on crypto. The official White House website (https://www.whitehouse.gov/briefing-room/statements-releases/2022/09/16/fact-sheet-white-house-releases-first-ever-comprehensive-framework-for-responsible-development-of-digital-assets/) affirms that the document intends to Protect Consumers, Investors, and Businesses. The document also shows the intention to reinforce U.S. financial leadership uphold U.S. values in global digital asset markets, and Explore a U.S. Central Bank Digital Currency (CBDC). This shows a will to keep the digital assets market under the control of the U.S. government, or at least more predictable.
+
+Regarding President Biden, there is a notable lack of interest in engaging with this particular segment of voters. This detachment is evident in the scarcity of interviews or public discussions where Biden directly addresses (positively or negatively) the cryptocurrency market or blockchain technology's implications for the future of finance and governance. 
```

Comment:
This is speculation. Find real evidence, official documents, that backup this speculation, one way or the other or lose the statement. Look at other people from his administration. To make this unbiased you should aim to get at least two pieces of evidence. Preferably a positive and a negative. But if all you can find is negative that’s also fine.

Reply:
I really haven't found quotes on Biden or people from the government talking directly about Bitcoin or other crypto. I really don't think it's the type of conversation that they engage. I'll let this statement out and talk about the latest actions regarding crypto